assignment7.txt

```diff
@@ -0,0 +1,14 @@
+Jack Rosen
+1. ++*p increments the value at point p. *p++ increments p and then returns the value. *++p increments p and then returns the value as well.
+2. It is only guaranteed for certain operators. The first level of operators goes from left to right, the second goes from right to left, and the rest go from left to right.
+3.The advantages of using pointers is that they are very powerful and help a person work with arrays. It is also a way to access a variable without having to call for it.
+4.1 char *
+4.2 invalid
```

Comment:
This is actually valid! "xyz" is just an array of characters, so the "xyz"[1] is just accessing "y". Then you just subtract 'y' from it to get 0. This is because, if you can recall, a char is just a value mapped to a character!